packages/react-native-editor/__device-tests__/gutenberg-editor-initial-html.test.js

```diff
@@ -16,22 +19,10 @@ describe( 'Gutenberg Editor Blocks test', () => {
 				lastBlockAccessibilityLabel
 			);
 		} else {
-			lastBlockElement = await editorPage.getLastElementByXPath(
-				lastBlockAccessibilityLabel
+			lastBlockElement = await editorPage.getBlockAtPosition(
```

Comment:
I noticed that `getBlockAtPosition` [has an option to auto-scroll to the element](https://github.com/WordPress/gutenberg/blob/fcaf478e3085ad7f0db4444de373ac919ae1f1dc/packages/react-native-editor/__device-tests__/pages/editor-page.js#L83), so we could use it also in the `if ( isAndroid() )` case to simplify the test, wdyt?

Reply:
> Thanks for the suggestion! I tried it but it is not working for this test, it begins to scroll down but then it stops just returning:
> 
> "//android.view.ViewGroup[contains(@content-desc, "Paragraph Block. Row 30")]" is still not visible after 25 retries!

Ah, it's a shame that didn't work. Thanks for checking it 🙇 !

> I'm going to merge this so we can see how it performs in trunk, if there's any tweaks needed I could investigate it further to unify it for both platforms.

Sure, my comment was just a suggestion, we can explore in the future enhancements to this test case.